### PR TITLE
Adop 208 birth mother details

### PR DESCRIPTION
**Pull request by abidfs**
### Change description
Added birth mother full name page
Added birth mother still-alive page
Added birth mother nationality page
Added birth mother occupation page
Added birth mother address known page
Added birth mother address pages (lookup, select, manual and international address)

Optimisation for full-name component
Optimisation for nationality component

### JIRA link
https://tools.hmcts.net/jira/browse/ADOP-401
https://tools.hmcts.net/jira/browse/ADOP-402

**Does this PR introduce a breaking change?** (check one with "x")
- [ ] Yes
- [x] No

### CODE REVIEW

File: src/main/steps/common/common.content.ts [hunk 1]
```diff
@@ -165,8 +166,9 @@ const cy: typeof en = {
     month: 'Mis',
     year: 'Blwyddyn',
   },
-  yes: 'Do',
```
Comment (renJosephHMCTS) [inline src/main/steps/common/common.content.ts:168]: Was this already approved Welsh lang? we could just add the Not Sure option if so

File: src/main/steps/task-list/utils.ts [hunk 2]
```diff
@@ -126,3 +129,38 @@ export const getAdoptionCertificateDetailsStatus = (userCase: CaseWithId): Secti
...
+
+  if (stillAlive === YesNoNotsure.NO) {
+    return names ? SectionStatus.COMPLETED : SectionStatus.IN_PROGRESS;
+  } else if (stillAlive === YesNoNotsure.NOT_SURE) {
+    const notAliveReason = userCase.birthMotherNotAliveReason;
+    return names && notAliveReason ? SectionStatus.COMPLETED : SectionStatus.IN_PROGRESS;
+  } else if (stillAlive === YesNoNotsure.YES) {
+    const nationality = userCase.birthMotherNationality;
+    const nationalities = userCase.birthMotherAdditionalNationalities;
+    //TODO consider not sure selection
```
Comment (renJosephHMCTS) [inline src/main/steps/task-list/utils.ts:145]: Please resolve this TODO

File: src/main/steps/common/common.content.ts (hunk 1 above)
Comment (abidfs) [reply src/main/steps/common/common.content.ts:168]: Yeah it was already approved welsh but text for some other application, hence I changed it to 'Yes (in welsh)',

File: src/main/steps/task-list/utils.ts (hunk 2 above)
Comment (abidfs) [reply src/main/steps/task-list/utils.ts:145]: hang on.. I need to handle this scenario..

Comment (abidfs) [reply src/main/steps/task-list/utils.ts]: handled it now.. please take a look

File: src/main/app/case/case.ts [hunk 3]
```diff
@@ -137,6 +138,22 @@ export interface Case {
...
+  birthMotherFirstNames?: string;
+  birthMotherLastNames?: string;
+  birthMotherStillAlive?: YesNoNotsure;
+  birthMotherNotAliveReason?: string;
+  birthMotherNationality?: string[];
+  birthMotherAdditionalNationalities?: string[];
+  birthMotherOccupation?: string;
+  birthMotherAddressKnown?: YesOrNo;
+  birthMotherAddress1?: string;
+  birthMotherAddress2?: string;
```
[the remark this answers is not in the document]
Comment (abidfs) [reply src/main/app/case/case.ts:150]: It's available
https://github.com/hmcts/adoption-web/blob/625c069525f6ba01c3048406b96e78d82cf9ee44/src/main/app/case/case.ts#L146

**Review by renJosephHMCTS — APPROVED** [review]

**Pull request merged**